code/obj/machinery/plantpot.dm

```diff
@@ -96,7 +96,7 @@
 	density = 1
 	mats = 2
 	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_CROWBAR
-	flags = NOSPLASH
+	flags = NOSPLASH | OPENCONTAINER
```

Comment:
```suggestion
	flags = NOSPLASH
```
moot if using smoke_start

Reply:
While it is no longer needed, the hydroponics try is still a container that is open. Im not sure if the tag messes something up, though it seems it doesnt. Though I have no problem removing it if needed.

Reply:
Actually the only way to get your hands on nectar was through smoking plant pots. I think this is just a fix to an oversight with the closed container change.